cli/exchange/node.go

```diff
@@ -656,6 +656,51 @@ func NodeListStatus(org string, credToUse string, node string) {
 
 }
 
+func NodeListNMPs(org, credToUse, nodeName string, all bool) {
+	msgPrinter := i18n.GetMessagePrinter()
+
+	cliutils.SetWhetherUsingApiKey(credToUse)
+	var nodeOrg string
+	nodeOrg, nodeName = cliutils.TrimOrg(org, nodeName)
+
```

Comment:
Make sure the node is registered. If not, the nmp list is empty. To check if the node is registered, get the node resource from the exchange, make sure `PublicKey` is not empty.